README.md

```diff
@@ -244,9 +244,12 @@ documentation][doc-logging] for more information.
 
 ## Compatibility
 
-- Apache Cassandra versions 2.1 and above.
-- DataStax Enterprise versions 4.8 and above.
-- Node.js versions 8 and above.
+The driver supports all versions of Node.js, Cassandra and DSE which are not EOL at the time of release.  For Node.js we support only
+LTS eligible branches (i.e. even numbered releases).  You can find more information about the Node.js release cycle in the [project documentation].  Consistent with this policy the current version of the driver offers support for the following:
```

Comment:
```suggestion
See the [project documentation] for more information about the Node.js release cycle. 
The current version of the driver offers support consistent with this policy for the following:
```

Reply:
I might wind up tweaking this one just slightly... let me see what it looks like in context.